### PR TITLE
Add rucio secondary software release policy

### PR DESCRIPTION
**Pull request by garciagenrique**
After discussion with the main developers of the applications
@ftorradeflot, any comment or modification is more than welcome.

Co-authored-by: maany <imptodefeat@gmail.com>

### CODE REVIEW

**Comment by ftorradeflot** [comment]
> After discussion with the main developers of the applications @ftorradeflot, any comment or modification is more than welcome.
> 
> Co-authored-by: maany [imptodefeat@gmail.com](mailto:imptodefeat@gmail.com)

Looks good to me. Thanks @garciagenrique !

File: docs/started/releasepolicy.md [hunk 1]
```diff
@@ -81,3 +81,53 @@ server.
...
+The plugin main dependencies are the Rucio REST API, the JupyterLab environment
+and JavaScript (Node.js and React.js). Up to date, the extension has been
+tested with several combinations of Rucio major versions (v1.30 onwards) and
+JupyterLab (v&GreaterEqual;3) environments, with the latter indicating the
+extension version to install.
+
+- For `JupyterLab v4.x`, use v&GreaterEqual;1.0.0.
+- For `JupyterLab v3.x` use the latest supported version (v0.10.0).
+
+Please refer to the Rucio JypyterLab extension [repository](
```
Comment (rdimaio) [inline docs/started/releasepolicy.md:131]:
```suggestion
Please refer to the Rucio JupyterLab extension [repository](
```

File: docs/started/releasepolicy.md [hunk 2]
```diff
@@ -81,3 +81,53 @@ server.
...
+   - **Minor Version**: Incremented for new features and improvements that are
+     backward-compatible.
+   - **Patch Version**: Incremented for backward-compatible bug fixes.
+
+### JupyterLab extension
+
+The Rucio JupyterLab extension does not follow the relase policy described
+above.
+
+The plugin main dependencies are the Rucio REST API, the JupyterLab environment
```
Comment (rdimaio) [inline docs/started/releasepolicy.md:122]:
```suggestion
The plugin's main dependencies are the Rucio REST API, the JupyterLab environment
```

**Review by rdimaio — CHANGES_REQUESTED** [review]

**Review by rdimaio — APPROVED** [review]

File: docs/started/releasepolicy.md [hunk 3]
```diff
@@ -81,3 +81,53 @@ server.
...
+## Secondary Rucio Software Policy
+
+### WebUI
+
+The Rucio WebUI is an initiative to modernize the Rucio user interface by
+leveraging the latest web technologies. Built with Next.js, TailwindCSS, and
+React.js, this new WebUI aims to provide a robust architecture designed to
+serve the community for the next decade, much like its predecessor.
+
+#### Key Technologies
```
Comment (bari12) [inline docs/started/releasepolicy.md:94]: Remove, no need to mention this here.

Also the text above, if we can just link to a rucio-webui page in the doc, we don't need to mention the entire text.

File: docs/started/releasepolicy.md [hunk 4]
```diff
@@ -81,3 +81,53 @@ server.
...
+
+- **Next.js**: Provides a flexible and efficient framework for server-side
+  rendering and static site generation.
+- **TailwindCSS**: Enables rapid and responsive UI development with
+  utility-first CSS.
+- **React.js**: Facilitates dynamic and interactive user interfaces with a
+  component-based approach.
+
+#### Versioning
```
Comment (bari12) [inline docs/started/releasepolicy.md:103]: Remove the section, just make it part of the section above

File: docs/started/releasepolicy.md [hunk 5]
```diff
@@ -81,3 +81,53 @@ server.
...
+
+The versioning of the Rucio WebUI is closely aligned with the Rucio Server's
+release versioning. Both follow semantic versioning principles, ensuring
+consistency and compatibility.
+
+- **Major Version**: The major version number of the WebUI matches that of the
+  Rucio Server. For example, Rucio Server version 35.x.x is compatible with
```
Comment (bari12) [inline docs/started/releasepolicy.md:110]: ... matches that of the Rucio server **it is compatible with**.

Comment (bari12) [inline docs/started/releasepolicy.md:110]: Also make the version examples bold (like above)

File: docs/started/releasepolicy.md [hunk 6]
```diff
@@ -81,3 +81,53 @@ server.
...
+  Rucio WebUI version 35.x.x.
+- **Minor and Patch Versions**: These versions follow semantic versioning:
+   - **Minor Version**: Incremented for new features and improvements that are
+     backward-compatible.
+   - **Patch Version**: Incremented for backward-compatible bug fixes.
+
+### JupyterLab extension
+
+The Rucio JupyterLab extension does not follow the relase policy described
```
Comment (bari12) [inline docs/started/releasepolicy.md:119]: Change to: The Rucio Jupyterlab extension follows it's own release policy described below.

File: docs/started/releasepolicy.md [hunk 7]
```diff
@@ -81,3 +81,53 @@ server.
...
+above.
+
+The plugin's main dependencies are the Rucio REST API, the JupyterLab environment
+and JavaScript (Node.js and React.js). Up to date, the extension has been
+tested with several combinations of Rucio major versions (v1.30 onwards) and
+JupyterLab (v&GreaterEqual;3) environments, with the latter indicating the
+extension version to install.
```
Comment (bari12) [inline docs/started/releasepolicy.md:126]: Maybe also mention that it will follow semantic versioning.

**Review by bari12 — CHANGES_REQUESTED** [review]
Thanks all. I have some minor adjustments :-)

**Review by bari12 — APPROVED** [review]
Thanks a lot everyone!

**Pull request merged**